activities/Chart.activity/NOTICE

```diff
@@ -0,0 +1,13 @@
+Copyright [yyyy] [name of copyright owner]
```

Comment:
To be compliant with the license, please write your name and the year in this file like here https://github.com/llaske/sugarizer/blob/dev/activities/MindMath.activity/NOTICE#L1